### ISSUE TITLE
FeCL loss

### ISSUE THREAD

**Issue by wjl991** [KU-CVML/DyCON#1]
In the paper, the FeCL loss is used for training on unlabeled data, so it should not involve the use of true labels, right? However, the paper employs true labels within this loss function. Is there a contradiction?

**Comment by Endarzboy** [comment]
Hi,

Yes, the FeCL loss is used for training the unalabled data, and the true labels are used to form a positve/negative pairs for the contrastive loss, the model didn't have the access of these labels. Note that, FeCL is built up on SupCon (a contrastive loss variant that empolys ground truth labels to form positive/negative pairs instead of relying on augmented views to form positive/negative pairs like SimCLR).

Thanks
